### ISSUE TITLE
fullpage.js with off-canvas navigation (mmenu.js)

### ISSUE THREAD

**Issue by kaspar-allenbach** [alvarotrigo/fullPage.js#227]
Hi. I try to implement mmenu.js (http://mmenu.frebsite.nl/tutorial.php) into fullpage.js

I almost works, but the anchor links inside the off-canvas menu aren't triggered. I guess this is because fullpage.js has its own way to handle the anchor links.
How do I pass along the anchor links from mmenu.js to fullpage.js ?

Thanks you.

You can see the almost working example here http://bit.ly/1fPpGp9

**Comment by eystein** [comment]
Hi. Bringing this back to live because I have the exact same problem and can't find a solution anywhere else. 

The javascript for the menu is in the `afterRender` callback, but to no avail. Everything works - except the links themselves, which don't even update the URL. I tried debugging with `console.log` within the callback, but got an `Uncaught SyntaxError: Unexpected token . `

**Comment by alvarotrigo** [comment]
@eystein I'm afraid that's a question for your menu plugin rather than for fullPage.js.
Moreover, I can not provide you with any help unless you provide us with a reproduction of your problem.

**Comment by eystein** [comment]
@alvarotrigo Thanks for the swift reply. 

I hacked together my own fix relying on `moveTo` in addition to the regular `anchors` based on [this issue comment](https://github.com/BeSite/jQuery.mmenu/issues/340#issuecomment-95963694) at the mmenu.js repository. 

After `$('#fullpage').fullpage({ ... });` I inserted:
```
// This works! (Repeat for each link) 
$('a[href="#about"]').click(function(e){
    e.preventDefault();
    $.fn.fullpage.moveTo(2, 0);
});
```
I don't understand why that makes any difference though. I tried to catch the click event from within `onclick`, but it didn't fire.
```
⋮
onclick: {
    preventDefault: function() {
        console.log('You clicked');
        return true;
    }
}
⋮
```
Hopefully this helps anyone with the same problem.
